pkg/tests/package_naming_aggregate_test.sh

```diff
@@ -0,0 +1,10 @@
+# This test simply checks that when we create packages with package_file_name
```

Comment:
Nits: Copyright?  Shebang line for `/bin/bash`?

Reply:
Done

Reply:
But I don't bother with the shebang. It is moot since we let the system shell win, which might b zsh on macos.